metadata.rb

```diff
@@ -9,6 +9,7 @@
 depends          'sudo'
 depends          'java'
 depends          'apache2'
+depends          'yum', '~> 3.10.0'
```

Comment:
just like how we do not have apt in as a dep that is an OS responsibility and don't want to require yum on a non centos/redhat system. Please remove this line.